Merge middle and tools, rename sections, make first more concise

SolalPirelli opened the pull request:
The first section, which is now "Getting started", is much more to the point and only keeps the subpages specific to the first year.

The middle section, now "Resources and responsibilities", includes the tools. (_The page about courses is severely lacking, but it's what it had before, and we'll improve that soon at the epic guide apéro Vini is planning_)

The last section, now "Graduating", has not changed.

shardulc commented:
Will review in detail in a bit, but as a quick high-level comment, I'm worried that removing the information about visas and residence permits will result in a lot of the same questions being asked on the IC Slack / similar venues. Yes, the information should all be available on official websites and is also pushed by EDIC admins, but people will still ask:
* I haven't gotten my {visa, permit appointment, biometric appointment, permit, attestation de domicile} yet! I'm worried! When will I get it?
* So far, I have done {steps}. Is the ball now in their court or still mine?
* My visa says "single entry". Does that mean I won't be able to reënter if I leave Switzerland?
  + (relatedly) My permit is expiring soon and I haven't gotten the renewed one yet and I have to leave Switzerland. What do I do?

SolalPirelli commented:
I removed it because it said practically nothing so I tried to summarize it within the bullet point in the checklist. This may be a good opportunity to expand on that.

Do you have answers for these questions? cause obviously I never had to deal with that here 😅

shardulc commented:
> * I haven't gotten my {visa, permit appointment, biometric appointment, permit, attestation de domicile} yet! I'm worried! When will I get it?
> * So far, I have done {steps}. Is the ball now in their court or still mine?

I think both types of questions can be answered by something like the following. Check the details though / feel free to edit before inclusion.

General protocol between you and various governments:
1. (non-EU nationals only) Apply for a long stay visa (90+ days) for Switzerland. The timeline, required paperwork, specific agencies you interact with, possibility of an interview, etc. depend heavily on your nationality. If you’re worried you may not have your visa in time for your start date at EPFL, contact EDIC.
2. (Lausanne residents only) [Pre-register](https://www.lausanne.ch/prestations/controle-des-habitants/etudiants-formalite-sejour.html) for a residence permit.
3. Arrive in Switzerland.
4. (Lausanne residents only) Wait for a *convocation* to arrive in the mail which will let you schedule an in-person appointment online and tell you what documents to bring. This should arrive approximately two weeks after the date you said you will arrive in the preregistration form.
5. (non-Lausanne residents only) Follow the instructions of your commune to announce your arrival (could be by email, postal mail, or an in-person visit) and apply for a residence permit.
6. When you submit the required paperwork for a permit, also request an *attestation de domicile / déclaration de résidence*. This is a temporary document that you can show to banks, rental agencies, etc. to prove that you have started the process for a residence permit even if you don’t have it yet. You should receive this immediately (if in-person) or in a few days (if by mail).
7. (non-EU nationals only) Wait for a *convocation* to arrive in the mail which will let you schedule an in-person appointment with the *Centre de biométrie*. Biometric data such as fingerprints and a photograph will be collected at this appointment and embedded in your permit card.
8. You will receive a letter confirming your personal information for the permit. No action is needed if the information is correct.
9. Your residence permit should arrive in the mail within three months. Actually, it is “registered mail”, so either you have to sign off on the delivery (which you most likely won’t, unless you work from home all the time), or the postal service will leave a ticket in your mailbox that you must present at your neighborhood post office to collect the delivery.
10. You may receive one or two invoices, perhaps before the actual permit. The total amount you will end up paying is up to 200 CHF depending on whether you asked for an *attestation de domicile*, whether you had to take a biometric appointment, and which commune you live in.
11. You will have to renew your permit every year (non-EU nationals) or every five years (EU nationals). Keep an eye out for a letter about the process a couple months before it expires, but if you don’t receive one, contact the *Contrôle des habitants* yourself.

Terminology:
* *Permis/titre/autorisation de séjour* = residence permit
* *Convocation* = summons, appointment invitation
* *État tiers* = non-EU/EFTA country
* *Attestation de domicile / déclaration de résidence* = residence certificate/proof (you have applied for a permit but don’t have one yet)
* *Centre de biométrie* = biometric center
* *Contrôle des habitants* = population office
* *Commune* = local administrative region, e.g. Lausanne, Ecublens, Renens, etc.

> * My visa says "single entry". Does that mean I won't be able to reënter if I leave Switzerland?
>    + (relatedly) My permit is expiring soon and I haven't gotten the renewed one yet and I have to leave Switzerland. What do I do?

With only a "single entry" visa and no residence permit, yes, you won't be able to reënter if you leave Switzerland. I’m not sure whether this includes the Schengen area or not. Once you have the residence permit, you can enter and leave Switzerland freely as long as the permit is valid. The visa no longer matters and immigration authorities won’t even look at it. If your permit is expiring soon and you have to leave Switzerland, you can get [a temporary return visa](https://www.vd.ch/themes/population/population-etrangere/entree-et-sejour/visa-de-retour-en-attente-du-renouvellement-du-titre-de-sejour-2) on short notice.

SolalPirelli commented:
Awesome! I added it as a new page and changed the checklist item to redirect to that. For consistency I added two checklist items for semester projects and for the candidacy exam.

shardulc commented on first.md:
```diff
@@ -1,32 +1,37 @@
+the EDIC staff for administrative questions,
+and of course the student representatives.
+
+You may also want to check out the "getting started" websites of
+the [EPFL IC doctoral school](https://www.epfl.ch/education/phd/edic-computer-and-communication-sciences/edic-computer-and-communication-sciences/edic-new-phd-students/),
+the [EPFL-wide doctoral school](http://phd.epfl.ch/new-students),
+and the [EPFL student affairs](https://www.epfl.ch/education/studies/en/).
```
Perhaps the ‘official resources’ should be recommended before our pages? But idk

shardulc commented on _first/legal.md:
```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "Legal formalities"
```
"Administrative" is a better description than "legal"

shardulc replied:
(change link to this page too)

shardulc reviewed (approved):
LGTM! A couple minor comments but you can merge whenever. Like I said, I’ll make a language/formatting pass later anyway.

SolalPirelli commented:
@shardulc re: the question on Slack right now about "annonce d'arrivée", are you sure the link you provided in point 2 of https://github.com/EPIC-guide/epic-guide.github.io/pull/42#issuecomment-1954386110 is the right one for PhD students? It seems designed for the undergrad kind of student.
